src/game/logic/object/object.cpp

```diff
@@ -23,10 +23,28 @@ ObjectShroudStatus Object::Get_Shrouded_Status(int index) const
     return ObjectShroudStatus(0);
 }
 
+// zh: 0x005479B0 wb: 0x007D0E3D
+Relationship Object::Get_Relationship(const Object *that) const
+{
+#ifdef GAME_DLL
+    return Call_Method<Relationship, const Object, const Object *>(PICK_ADDRESS(0x005479B0, 0x007D0E3D), this, that);
+#else
+    return Relationship();
+#endif
+}
+
 Player *Object::Get_Controlling_Player() const
 {
 #ifdef GAME_DLL
     return Call_Method<Player *, const Object>(PICK_ADDRESS(0x00547A00, 0x007D0EAB), this);
 #endif
     return nullptr;
 }
+
+// zh: 0x00549BD0 wb: 0x007D370C
+void Object::On_Veterancy_Level_Changed(VeterancyLevel v1, VeterancyLevel v2, bool b)
+{
+#ifdef GAME_DLL
+    Call_Method<void, Object, VeterancyLevel, VeterancyLevel, bool>(PICK_ADDRESS(0x00549BD0, 0x007D370C), this, v1, v2, b);
+#endif
```

Comment:
This function is missing code for Standalone? Maybe at least add a `#pragma message` clarifying this here.

Reply:
This is the style of all functions that are unimplemented